app/experimenter/experiments/models.py

```diff
@@ -505,21 +505,32 @@ def is_paused_published(self):
 
     @property
     def monitoring_dashboard_url(self):
-        start_date = (self.start_date or datetime.date.today()) - datetime.timedelta(
-            days=1
-        )
+        if not self.is_rollout:
```

Comment:
Oh no no no, experiments and rollouts both have monitoring for enrollment.  It's only the opmon link that needs to be guarded.